projects/core/styles/basic/normalize.less

```diff
@@ -69,7 +69,7 @@ main {
  */
 
 figure {
-    margin: 1em 40px;
+    margin: 1em 2.5rem;
 }
 
```

Comment:
Please pay attention to this file, shoul we switch to rem outline-offset?
`
[type='search'] {    -webkit-appearance: textfield; /* 1 */    outline-offset: -2px; /* 2 */}
`

Reply:
I don't think it is relevant but let's keep it in px :)